isis/src/control/apps/jigsaw/tsts/scconfigHeld/Makefile

```diff
@@ -13,9 +13,18 @@ commands:
 	  bundleout_txt=no \
 	  output_csv=no \
 	  residuals_csv=no > /dev/null;
-	$(CAT) bundleout_images.csv \
-	  | perl -pe 's/(^|,|: )([^,:]+\/)([^,\/:]*\.)(net|cub)/\1\3\4/g' 2> /dev/null \
-	  > $(OUTPUT)/bundleout_scconfig_images.csv;
+	$(CAT) bundleout_images_MARS_ODYSSEY_THEMIS_IR.csv \
+	   | perl -pe 's/(^|,|: )([^,:]+\/)([^,\/:]*\.)(net|cub)/\1\3\4/g' 2> /dev/null \
+	   > $(OUTPUT)/bundleout_scconfig_images_themis.csv;
+	$(CAT) bundleout_images_VIKING_ORBITER_1_VISUAL_IMAGING_SUBSYSTEM_CAMERA_B.csv \
+	   | perl -pe 's/(^|,|: )([^,:]+\/)([^,\/:]*\.)(net|cub)/\1\3\4/g' 2> /dev/null \
+	   > $(OUTPUT)/bundleout_scconfig_images_viking_1.csv;
+	$(CAT) bundleout_images_VIKING_ORBITER_2_VISUAL_IMAGING_SUBSYSTEM_CAMERA_A.csv \
+          | perl -pe 's/(^|,|: )([^,:]+\/)([^,\/:]*\.)(net|cub)/\1\3\4/g' 2> /dev/null \
+          > $(OUTPUT)/bundleout_scconfig_images_viking_2.csv;
+	$(CAT) bundleout_images_held.csv \
+          | perl -pe 's/(^|,|: )([^,:]+\/)([^,\/:]*\.)(net|cub)/\1\3\4/g' 2> /dev/null \
+          > $(OUTPUT)/bundleout_scconfig_images_held.csv;
```

Comment:
This is a good example of what the new output can look like. This test has THEMIS images, Viking 1 images, VIking 2 images, and a held image.